src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -253,7 +255,7 @@ public class CRLP_RollupContact_TEST {
      * 4. Nightly LDV Batch Job
      */
     private static void testRollupsServices(TestType tt) {
-
+        Map<Id, Contact> mapConIdCon;
```

Comment:
> Let's not change existing tests.

I noticed that it is hitting SOQL query limitations when I added a query before in the same method. I thought this will reduce the number of SOQL queries in that method and want to keep it, let me know your thoughts.

Reply:
Did the test fail due to SOQL limit being reached? If not, let's keep the test the same to minimize the  number of moving parts. If there is need to change this, we can fix it in another WI. Thoughts?

Reply:
The tests failed when I added one more query in the method (it said reached 101 queries in that method).